media-libs/imlib2/imlib2-1.4.9.recipe

```diff
@@ -0,0 +1,115 @@
+SUMMARY="An advanced replacement library for libraries like libXpm"
+DESCRIPTION="A library that does image file loading and \
+saving as well as rendering, manipulation, arbitrary polygon support, etc.
+
+It does ALL of these operations FAST. Imlib2 also tries to be highly \
+intelligent about doing them, so writing naive programs can be done \
+easily, without sacrificing speed.
+
+This is a complete rewrite over the Imlib 1.x series. The architecture is \
+more modular, simple, and flexible. See index.html in the doc/ directory \
+for more information."
+HOMEPAGE="https://www.enlightenment.org/"
+COPYRIGHT="2002 Project Enlightenment"
+LICENSE="IMLIB2"
+SOURCE_URI="http://downloads.sourceforge.net/enlightenment/imlib2-$portVersion.tar.bz2"
+CHECKSUM_SHA256="7d2864972801823ce44ca8d5584a67a88f0e54e2bf47fa8cf4a514317b4f0021"
+REVISION="1"
+
+ARCHITECTURES="?x86_gcc2 x86 ?x86_64"
```

Comment:
It does not build on x86_gcc2, and it builds fine on x86_64. So you might wish to change this line to:
```
ARCHITECTURES="!x86_gcc2 x86 x86_64"
```
BTW, there are a few other cosmetic changes that you might wish to copy from https://github.com/haikuports/haikuports/compare/master...fbrosson:imlib2 :
* DESCRIPTION: optimized line wrapping;
* TEST: Add function definition with the generic "make check".

In case you would like try the "[Allow edits from maintainers](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/)" feature recently added by GitHub, please feel free to do so. My remote branch is ready for a push to this PR ;-)